src/pages/Dashboard/TeamOverview.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { Box, Button, Center, Spacer, Heading } from '@chakra-ui/react';
+
+const GET_TEAM_INFO = gql`
+  query getTeam($id: Int!) {
+    getTeam(id: $id) {
+      name
+      organization
+      id
+      amountRaised
+      itemsSold
+    }
+  }
+`;
+
+const StorefrontButton = () => {
+  return <Button w="199px">Share Storefront</Button>;
+};
+
+const SalesInfo = props => {
+  return (
+    <Box mr="100px">
+      <Heading fontSize="h4" color="gray.400" mb="8px">
+        {props.description}
+      </Heading>
+      <Heading fontSize="32px">{props.amount}</Heading>
+    </Box>
+  );
+};
+
+const NoSales = () => {
+  return (
+    <Box w="100%" h="300px" bg="background.primary">
+      <Center w="100%">
+        <Heading fontSize="20px" mt="69px" mb="24px">
+          You don’t have sales yet. Share your storelink to get started!
+        </Heading>
+      </Center>
+      <Center w="100%">
+        <StorefrontButton />
+      </Center>
+    </Box>
+  );
+};
+
+const TeamOverview = ({ teamId }) => {
+  const { loading, error, data } = useQuery(GET_TEAM_INFO, {
+    variables: { id: teamId },
+  });
+
+  return loading ? (
+    'Loading...'
+  ) : error ? (
+    `Error! ${error.message}`
+  ) : (
+    <Box w="100%">
+      <Heading fontSize="h4" color="gray.400" mb="8px">
```

Comment:
There are custom styles available for `Heading`. You can use them by setting the `size` prop. It would be good to use them to keep the UI looking consistent

Reply:
see `ChakraExpoDashboard.js` for more ref, I believe this gray one you would do `size="subtitle"